WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,228 @@
+
+After the WEP is ready for the community review, the reviewer assigns the **ready-for-discussion** label to the PR. The CoreOps team is then responsible for distributing announcements of each WEP to the relevant community segment. For example, a WEP about meetups will be shared with the Meetup team for distribution between the meetup organizers. For some WEPs, community-wide announcements will be made on Slack or to the mailing list. 
+  
+Community review, discussion, and updates
+.........................................
+
+The standard community review period for a WEP is **30 days**. The WEP is reviewed by a process called **lazy consensus**. This means that community members either provide feedback, approve the WEP explicitly, or ignore the WEP. 
+
+If the review period ends without a clear consensus, but further discussion is still needed, the review period might be extended by the WEP team. The review period is never shorter than one month, but other WEPs can be submitted in the meantime, and multiple WEPs might be undergoing a community review simultaneously. 
+
+During the community review period, the WEP goes through one or more iterations of feedback and updates. Community feedback is submitted in the form of comments to the PR, and the WEP author is responsible to update the WEP content in case the incoming feedback is accepted.
+
+All discussion on the WEP must happen on the PR in the form of comments in order to be considered as official feedback on the WEP. Of course external conversations can happen on other platforms, but the official record for a WEP discussion is only on the PR itself. If discussion happens on another platform, we recommend copying the most important points or the results of those discussions into the WEP comments.
+
+You must have a GitHub account to post comments to a PR. If you need help getting started with GitHub, email support@writethedocs and the WTD team will help you set up your account and get familiar with GitHub basics. 
+
+Final review and resolution
+...........................
+
+At the end of the community review period, the WEP is accepted if it meets the minimum criteria and received lazy consensus as described in the previous process stages.
+
+The WEP team reviews the final WEP content and if it is accepted, the team merges the PR, removes the **ready-for-discussion** flag, and moves the WEP to the ``accepted`` directory of the GitHub repository. 
+
+Possible resolutions for a WEP are:
+
+Accepted
+  Reason: Community feedback was completed with lazy consensus achieved, and the WEP content is complete and follows the formatting guidelines. 
+  Action: The WEP team merges the PR and stores the WEP in the ``accepted`` directory.
+
+Rejected
+  Reason: Community feedback led to disagreements about the WEP that cannot be resolved, or the original goal of the WEP is no longer considered an improvement to the community. 
```

Comment:
> no longer considered an improvement to the community

I think the passive voice here makes it hard to understand the process. Is this meant to be a characterization of the review discussion (e.g., community feedback leads to a negative consensus that the WEP should be rejected)? Or is this an independent conclusion of the WEP team (e.g., the WEP team reviews the final text and makes its own decision on whether its an improvement to the community)?

Reply:
The former, I'd say.

Reply:
Agreed that I think we meant the former. I guess we should have a state that is "rejected by the powers that be" -- perhaps `Vetoed`, which makes this more explicit?

Reply:
I'm not comfortable with a Veto option here, since it seems a bit contradictory to the consensus idea. I made a suggestion to reword the text to make it more explicit, maybe it'll solve the question.